sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -709,9 +711,12 @@ public Key restoreKey(byte[] backup) {
      * call {@link KeyClient#getKey(KeyBase baseKey)} . This will return the {@link Key key} with key material included of its latest version.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listKeys}
      *
-     * @return A {@link List} containing {@link KeyBase key} of all the keys in the vault.
+     * <p><strong>Samples to iterate KeyBase by each page</strong></p>
+     * {@codesnippet com.azure.keyvault.keys.keyclient.listKeys.iterableByPage}
+     *
+     * @return {@link PagedIterable} of {@link KeyBase key} of all the keys in the vault.
```

Comment:
Rename the link to KeyBases not key, shouldn't be hiding the class name here.

Reply:
The original documentation in whole class is using key instead of KeyBase. If we change here, we will have to change all other documentation as well.

Reply:
Yeah, key base should be represented as keys to the user for this method.